rpmbuild/bin/copr-builder

```diff
@@ -0,0 +1,222 @@
+#! /usr/bin/python3
+
+"""
+Allow users to control their Copr builder instance
+https://github.com/fedora-copr/debate/tree/main/user-ssh-builders
+"""
+
+import argparse
+from datetime import datetime, timedelta
+from contextlib import suppress
+from copr_common.helpers import timedelta_to_dhms
+
+
+RPMBUILD_PID_PATH = "/var/lib/copr-rpmbuild/pid"
+EXPIRATION_PATH = "/run/copr-builder-expiration"
+
+# See also background_worker_build.py
+DEFAULT_EXPIRATION = 60 * 60
+MAX_EXPIRATION = 60 * 60 * 24 * 2
+
+
+def cmd_help():
+    """
+    Print full instructions for working with the builder instance.
+    Ideally we would have this in MOTD but that would cause problems in cases
+    like `if $(ssh stroj cat foo|head -n1) == "foo"`. So instead, we instruct
+    users to run `copr-builder help` manually.
+    """
+    # TODO Should this be in the documentation instead?
+    print(
+        "You have been entrusted with root access to a Copr builder.\n"
+        "Please be responsible.\n"
+        "\n"
+        "This instance should be used only for the purposes of debugging\n"
+        "build failures of RPM packages. It is not a personal VPS :-).\n"
```

Comment:
I think we could be more "strict", without a joking tone, ala motd on Batcave:
```
This is a PRIVATE computer system, unauthorized access is
strictly prohibited.

This system is to be used for Copr and related purposes only.  This
is not your personal computing system.  Users who are unsure whether or not
they have access to this system, don't have it and should log off immediately.
```

Reply:
Updated. I tried some middle ground between my half-joking tone and your super strict one :-)
I don't want to be too strict and provoke people to do bad stuff just to try if they can get away with it.